frontend/packages/ceph-storage-plugin/src/components/block-pool/block-pool-menu-action.ts

```diff
@@ -0,0 +1,28 @@
+import { TFunction } from 'i18next';
+
+import { asAccessReview } from '@console/internal/components/utils';
+import { K8sKind } from '@console/internal/module/k8s';
+
+import { StoragePoolKind } from '../../types';
+import { updateBlockPoolModal } from '../modals/block-pool-modal/update-block-pool-modal';
+import { deleteBlockPoolModal } from '../modals/block-pool-modal/delete-block-pool-modal';
+
+const editBlockPool = (kindObj: K8sKind, blockPoolConfig: StoragePoolKind, _, customData) => {
```

Comment:
ResourceKabab will call this function like
 actions.map((a) => a(kindObj, resource, null, customData))
 
 3rd one is always null